docs/guide/quick-start.md

```diff
@@ -73,9 +72,8 @@ class AuthHandler
         if (Yii::$app->user->isGuest) {
             if ($auth) { // login
                 /* @var User $user */
-                $user = $auth->user;
-                $this->updateUserInfo($user);
-                Yii::$app->user->login($user, Yii::$app->params['user.rememberMeDuration']);
+                $user = User::findIdentity($auth->user_id);
+                Yii::$app->user->login($user);
```

Comment:
Why `Yii::$app->params['user.rememberMeDuration']` was removed?

Reply:
because default configuration in web.php (yii2-basic) doesn't have params['user.rememberMeDuration']. Is it mean I add params['user.rememberMeDuration'] in configuration file (web.php if yii2-basic)?

Reply:
The name is very descriptive and the error about it missing should be enough to add it.